src/components/ListItem.js

```diff
@@ -43,10 +40,11 @@ const ListItem = () => {
         onSnapshot(doc(itemsCollectionRef, currToken), (doc) => {
           if (!doc.data()) {
             setEmptyList(true);
+            setLoading(false);
           } else {
             setItems(doc.data().items);
+            setLoading(false);
```

Comment:
This is an interesting change, the old code should have the same functionality, I'm curious, what problem were you solving with this change?